docs/en/contribution-roadmap/Knowledge.md

```diff
@@ -10,6 +10,35 @@ priority: 10
 
 ## Java
 
+This roadmap assumes you have the basic knowledge about Java:
+* You understand the Java basics e.g. classes, objects, primitives, references, methods, 
```

Comment:
use `basic` too much around here, which makes it sound like all these things should be super easy and can discourage new people. reword to `You understand Java constructs such as classes, objects, ...`